src/components/projects/lean-workflow/ManualFlowTransitioning.tsx

```diff
@@ -20,9 +20,11 @@ import {
   TaskStepDoneIcon,
   usePermissions,
 } from '@manuscripts/style-guide'
+import { trackCommands } from '@manuscripts/track-changes-plugin'
```

Comment:
hm, I thought we are trying to move away from using that component. Am I confusing it with something?

Reply:
I can't remember any discussion around that 🤔